src/Ancestor/Ancestor.DataAccess/3.DAO/OracleDao.cs

```diff
@@ -17,7 +17,8 @@ namespace Ancestor.DataAccess.DAO
     // 2015-10-22 1. GetDbType() 加入 SYSTEM.DATETIME 型態的轉換
     // 2016-02-08 1. Add feature for Dispose. 2. 修改取得DbType的方法, 改為 Dictionary 方式取得
     // 2016-04-05 Add feature SaveChange and CancellChange for transaction.
-    public class OracleDao : DataAccessObject, IDataAccessObject
+    // 2017-07-13 修正為繼承BasweAbstractDao
+    public class OracleDao : BaseAbstractDao, IDataAccessObject
```

Comment:
Because BaseAbstractDao was already implement IDataAccessObject
So IDataAccessObject can be removed from here.